dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,108 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Check support for rsa2k signed certificates
```

Comment:
@tym2k1 please add short test description for every test case similar to https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/dasharo-security/secure-boot.robot#L185; explain there how we are creating the given keys

Reply:
Addressed and squashed